contracts/apps/disputable/DisputableAragonApp.sol

```diff
@@ -97,28 +89,54 @@ contract DisputableAragonApp is IDisputable, AragonApp {
         return _getAgreement();
     }
 
+    /**
+    * @dev Challenge disputable action
+    * @param _disputableActionId Identification number of the disputable action to be challenged
+    * @param _challengeId Identification number of the challenge in the context of the Agreement
+    * @param _challenger Address challenging the disputable
+    */
+    function _onDisputableActionChallenged(uint256 _disputableActionId, uint256 _challengeId, address _challenger) internal;
+
+    /**
+    * @dev Reject disputable action
+    * @param _disputableActionId Identification number of the disputable action to be rejected
+    */
+    function _onDisputableActionRejected(uint256 _disputableActionId) internal;
+
+    /**
+    * @dev Allow disputable action
+    * @param _disputableActionId Identification number of the disputable action to be allowed
+    */
+    function _onDisputableActionAllowed(uint256 _disputableActionId) internal;
+
+    /**
+    * @dev Void disputable action
+    * @param _disputableActionId Identification number of the disputable action to be voided
+    */
+    function _onDisputableActionVoided(uint256 _disputableActionId) internal;
+
     /**
     * @dev Create a new action in the agreement without lifetime set
     * @param _disputableActionId Identification number of the disputable action in the context of the disputable
     * @param _submitter Address of the user that has submitted the action
     * @param _context Link to a human-readable text giving context for the given action
```

Comment:
If we keep this overload, let's remember to update the natspec :)

Although I'm surprised this even compiles; solidity usually errors if the natspec is wrong.

Reply:
I don't think the compiler fails if the order is not correct